src/features/ITGovernance/Admin/GRBReview/GRBReviewForm/Participants/index.tsx

```diff
@@ -1,26 +1,344 @@
+} from 'gql/generated/graphql';
 
+import DatePickerFormatted from 'components/DatePickerFormatted';
 import { EasiFormProvider, useEasiForm } from 'components/EasiForm';
+import FieldErrorMsg from 'components/FieldErrorMsg';
+import Modal from 'components/Modal';
+import RequiredAsterisk from 'components/RequiredAsterisk';
+import useMessage from 'hooks/useMessage';
 import { GRBReviewFormStepProps } from 'types/grbReview';
+import { SetGRBParticipantsAsyncSchema } from 'validations/grbReviewSchema';
 
+import ParticipantsTable from '../../ParticipantsSection/_components/ParticipantsTable';
 import GRBReviewFormStepWrapper, {
   GRBReviewFormStepSubmit
 } from '../GRBReviewFormStepWrapper';
 
-// TODO: Update fields type
-type ParticipantsFields = {};
+type ParticipantsFields = {
+  grbReviewers: SystemIntakeGRBReviewerFragment[];
+  grbReviewAsyncEndDate: SystemIntakeGRBReviewFragment['grbReviewAsyncEndDate'];
+};
 
 const Participants = ({ grbReview }: GRBReviewFormStepProps) => {
-  const form = useEasiForm<ParticipantsFields>();
+  const { t } = useTranslation('grbReview');
+
+  const [mutate] =
+    useUpdateSystemIntakeGRBReviewFormInputTimeframeAsyncMutation();
+
+  const form = useEasiForm<ParticipantsFields>({
+    resolver: yupResolver(SetGRBParticipantsAsyncSchema),
+    defaultValues: {
+      grbReviewers: grbReview.grbReviewers,
+      grbReviewAsyncEndDate: grbReview.grbReviewAsyncEndDate || ''
+    },
+    values: {
+      grbReviewers: grbReview.grbReviewers,
+      grbReviewAsyncEndDate: grbReview.grbReviewAsyncEndDate || ''
+    }
+  });
+  const {
+    control,
+    formState: { errors }
+  } = form;
+
+  const onSubmit: GRBReviewFormStepSubmit<ParticipantsFields> = async input => {
+    return mutate({
+      variables: {
+        input: {
+          systemIntakeID: grbReview.id,
+          grbReviewAsyncEndDate: input.grbReviewAsyncEndDate ?? ''
+        }
+      }
+    });
+  };
+
+  const history = useHistory();
+  const { pathname } = useLocation();
+  const { showMessage } = useMessage();
+
+  const [reviewerToRemove, setReviewerToRemove] =
+    useState<SystemIntakeGRBReviewerFragment | null>(null);
+
+  const [deleteReviewer] = useDeleteSystemIntakeGRBReviewerMutation({
+    refetchQueries: [GetSystemIntakeGRBReviewDocument]
+  });
+
+  const removeGRBReviewer = useCallback(
+    (reviewer: SystemIntakeGRBReviewerFragment) => {
+      deleteReviewer({ variables: { input: { reviewerID: reviewer.id } } })
+        .then(() =>
+          showMessage(
+            <Trans
+              i18nKey="grbReview:messages.success.remove"
+              values={{ commonName: reviewer.userAccount.commonName }}
+            />,
+            { type: 'success' }
+          )
+        )
+        .catch(() =>
+          showMessage(t('form.messages.error.remove'), { type: 'error' })
+        );
+
+      // Reset `reviewerToRemove` to close modal
+      setReviewerToRemove(null);
+    },
+    [deleteReviewer, showMessage, t]
+  );
```

Comment:
Is there a reason why you are using `useCallback` here?  I don't think it's needed.  It's being called on click and not in within some hook

Reply:
hm! sorry it was a simple copy and paste from `src/features/ITGovernance/Admin/GRBReview/index.tsx` where the same `removeGRBReviewer` functionality is used. 
Converting the one here to be a simple onclick function